xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_version.xsd

```diff
@@ -175,9 +175,9 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>PASSING TIMEs  for JOURNEY.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="datedCalls" type="datedCalls_RelStructure" minOccurs="0">
+			<xsd:element name="datedCalls" type="calls_RelStructure" minOccurs="0">
```

Comment:
I guess the name has been left unchanged deliberataly, to ensure compatibility.

Reply:
They are still dated (since in DatedVJ) but can use the traditional Call structure (no need for a dated version)